common/models/application.js

```diff
@@ -71,13 +72,11 @@ function generateKey(hmacKey, algorithm, encoding) {
 
 module.exports = function(Application) {
   // Workaround for https://github.com/strongloop/loopback/issues/292
-  Application.definition.rawProperties.created.default =
```

Comment:
it was giving this error:
```
>> Mocha exploded!
>> TypeError: Cannot create property 'default' on string 'date'
>>   at module.exports (/Users/loay/Desktop/NodeEE/loopback/common/models/application.js:75:56)
```

As you see, I removed the first line in `application.js` and `role.js` but I think the proper solution is in `application.json` and `role.json`

```
    "created": {
      "default": "date"
    },
    "modified": {
      "default": "date"
    }
```
I will update that in the next commit.